Makefile

```diff
@@ -176,19 +176,19 @@ copy-files:
 	install -d -m 755 $(DESTDIR)/srv/salt/ceph/functests/1node/tuned/off
```

Comment:
Could you fix the commit title? (Should be "move 1node.migrate to 3nodes.migrate", I guess.)

Reply:
err, yes